$OpenBSD: patch-gnulib_m4_manywarnings_m4,v 1.1 2012/09/11 10:51:13 jasper Exp $

Remove -Wredundant-decls; the time.h and sys/time.h headers from
gnulib are causing a metric ton of (invalid) warnings that hide
real warning from the logfiles.

--- gnulib/m4/manywarnings.m4.orig	Tue Sep 11 11:55:12 2012
+++ gnulib/m4/manywarnings.m4	Tue Sep 11 11:55:20 2012
@@ -120,7 +120,6 @@ AC_DEFUN([gl_MANYWARN_ALL_GCC],
     -Wmissing-format-attribute \
     -Wpacked \
     -Wpadded \
-    -Wredundant-decls \
     -Wnested-externs \
     -Wunreachable-code \
     -Winline \
